pyproject.toml

```diff
@@ -37,7 +37,7 @@ dependencies = [
     "XStatic-asciinema-player <= 2.6.1.1",
     "xstatic-bootbox >= 5.4.0, <= 5.5.1.1",
     "xstatic-bootstrap >=4.0.0.0, <= 4.5.3.1",
-    "xstatic-font-awesome <= 4.7.0.0",
+    "xstatic-font-awesome >= 6.0.0, <= 6.2.1.1",
```

Comment:
Note: 6.2.1.1 is the latest fa xstatic package at the time of writing. it also might be the last "legacy" fa xstatic package, before xstatic gets modernized.